tests/test_base.py

```diff
@@ -87,9 +88,10 @@ def test_load_all_packages(component_type: PackageType, config_file_path: str) -
     )
     with open_file(config_file_path) as fp:
         configuration_object = configuration_loader.load(fp)
-        directory = Path(config_file_path).parent
-        configuration_object.directory = directory
-        load_aea_package(configuration_object)
+        if component_type != PackageType.SERVICE:
+            directory = Path(config_file_path).parent
+            configuration_object.directory = directory
+            load_aea_package(configuration_object)
```

Comment:
I think this line should not be nested?

Why are services not being assigned a directory?

Reply:
Sorry, for not providing context. The nested logic also loads dependency packages which is not required in service component. We only want to check if we can load a service component config without any error.

Reply:
This comment should really be there in code. I will merge into the base branch as you need to fix things there but please add it.